hack/build.sh

```diff
@@ -22,6 +22,13 @@ TAGS="${TAGS:-}"
 OUTPUT="${OUTPUT:-bin/openshift-install}"
 export CGO_ENABLED=0
 
+if (echo "${TAGS}" | grep -q 'okd')
+then
+	# Replace rhcos images with FCOS
+	mv ./data/data/{fcos,rhcos}.json
+	mv ./data/data/{fcos,rhcos}-amd64.json
```

Comment:
This is not acceptable. This will destructively change the files in the source tree. If a user builds once with the okd tag, then subsequent builds without the okd tag will use the fcos image incorrectly.

Reply:
I used a more invasive solution in https://github.com/vrutkovs/installer/commit/e69ce98333844d4d8b4b57ba2dc8334362f1770c - does it look better?

Reply:
What are we gaining by plumbing `isOKD` through all of the function calls? Instead of relying on pushing the results of `(*InstallConfig).IsOKD` around everywhere, can we have `fetchRHCOSBuild` get the value directly from the public `OKD` variable in the types pacakge?

Reply:
Reworked this to use CoreOS streams